hawkular-alerts-engine/src/test/java/org/hawkular/alerts/engine/PersistenceTest.java

```diff
@@ -2380,4 +2379,108 @@ public void test0140EventTags() throws Exception {
         assertEquals("EventTag2Value", event.getTags().get("EventTag2Name"));
     }
 
+    // These tests would be nice in a separate clas but I couldn't figure how to get multiple test classes
+    // to run embedded Cassandra without stepping on each other.
+
+    private long nqs100Time = 0L;
+
+    @Test
+    public void test0200GetEventsNQS100() throws Exception {
+        EventsCriteria criteria = new EventsCriteria();
+        criteria.setEndTime(50L);
+        criteria.setCategory("category-a");
+        criteria.setCriteriaNoQuerySize(100);
```

Comment:
Good comment.  At the moment it is an internal helper (good for testing) because I did not add it to the REST API, but it perhaps could be added to give users control if they were to need it.  There is also the new sysprop for global override, exposing this would also allow for fine-grained override.  We could add later to REST if it seems like a good idea.